.github/workflows/blocked-reminder.yaml

```diff
@@ -79,7 +79,7 @@ jobs:
         uses: archive/github-actions-slack@d9dae40827adf93bddf939db6552d1e392259d7d # v2.7.0
         if: ${{ steps.list-issues.outputs.result != '' }}
         with:
-          slack-bot-user-oauth-access-token: ${{ secrets.WEAVEWORKS_SLACK_GENERICBOT_TOKEN }}
```

Comment:
@chanwit should this be reverted or fixed in the repo secrets?

Reply:
Please remove the whole `.github/workflows/blocked-reminder.yaml` file.
It was for an internal team to get notified about blocking issues.